rotkehlchen/accounting/structures/base.py

```diff
@@ -331,7 +281,7 @@ def process(
             )
             return 1
         # else it's a decoded transaction event and should be processed there
-        return accounting.transactions.process(self, events_iterator)
+        return accounting.transactions.process(self, events_iterator)  # type: ignore[arg-type]
```

Comment:
`self` here is a `HistoryBaseEntry` while `transacions.process` expects an `EvmEvent`.

Can't use `cast` here since `cast` requires the type to be provided. And we can't import `EvmEvent` here because we will get recursive imports.

Reply:
okey but add it as a comment to the code. Is what we do always that we ignore a type